tutorials/ten_minutes.ipynb

```diff
@@ -10,9 +10,9 @@
     "\n",
```

Comment:
<p>This looks good. There just seems to be an extra new line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I'm not sure what is going on here. There is no extra new-line on my side. This most be a rendering bug on ReviewNB 's behalf.</p>

Reply:
Ok that's fine don't worry about it. We can fix it anyway later if it persists.